packages/payload/src/admin/components/forms/field-types/Array/index.tsx

```diff
@@ -35,11 +35,11 @@ const ArrayFieldType: React.FC<Props> = (props) => {
     forceRender = false,
     indexPath,
     localized,
+    required,
+    minRows = required ? 1 : 0,
```

Comment:
if `required` is `true` but no value has been passed to `minRows`, set a fallback value so that it is not `undefined`